src/views/catalog/wizard/tabs/network/components/modal/NetworkInterfaceModal.tsx

```diff
@@ -1,17 +1,12 @@
-import * as React from 'react';
+import React, { useCallback } from 'react';
 
 import { produceVMNetworks, UpdateValidatedVM } from '@catalog/utils/WizardVMContext';
-import { V1Interface, V1Network, V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
-import TabModal from '@kubevirt-utils/components/TabModal/TabModal';
-import { Form } from '@patternfly/react-core';
-
-import NameFormField from './components/NameFormField';
-import NetworkInterfaceMACAddressInput from './components/NetworkInterfaceMacAddressInput';
-import NetworkInterfaceModelSelect from './components/NetworkInterfaceModelSelect';
-import NetworkInterfaceNetworkSelect from './components/NetworkInterfaceNetworkSelect';
-import NetworkInterfaceTypeSelect from './components/NetworkInterfaceTypeSelect';
-import { interfaceModelType, interfaceTypeTypes } from './utils/constants';
-import { generateNicName, networkNameStartWithPod, podNetworkExists } from './utils/helpers';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import SharedNetworkInterfaceModal from '@kubevirt-utils/components/NetworkInterfaceModal/NetworkInterfaceModal';
```

Comment:
suggestion: It's a bit confusing to me when here you call the shared component SharedNetworkInterfaceModal and in the Edit just NetworkInterfaceModal, can you just call SharedNetworkInterfaceModal to all of them? also maybe a better name than Shared... ?

Reply:
I thought i would get away with it, lol. I didn't have a better naming... i will think about something :)

Reply:
done - i took a different approach instead of renaming the shared one, I renamed the calling components to match its place of rendering name

Reply:
I secretly hoped that's what you'll do :) 
great minds think alike